### PR TITLE
Open source health design

### PR DESCRIPTION
**Pull request by Ines-C-Amri**
I created a new Open Source page with the new designs --this is just the start

### CODE REVIEW

File: src/pages/open-source-health-design.js [hunk 1]
```diff
@@ -0,0 +1,949 @@
...
+          </div>
+
+          <div className="pad-vertical ">
+            <h2 style={headerStyleCentered}>
+              We envision a world where patients can be cared for with a
+              technology that can be trusted.
+            </h2>
+            <p id="open-work" style={npStyleCentered}>
+              For the past 10 years, we’ve partner with Federal and Non-profit
+              organizations to design more than 60 open source projects to the
```
Comment (ericbenwa) [inline src/pages/open-source-health-design.js:225]: "we've partnered..."

"Nonprofit"... hyphenate? the wiki article doesn't.
https://en.wikipedia.org/wiki/Nonprofit_organization

File: src/pages/open-source-health-design.js [hunk 2]
```diff
@@ -0,0 +1,949 @@
...
+              solve most pressing healthcare issues.
```
Comment (ericbenwa) [inline src/pages/open-source-health-design.js:226]: "to solve..."
currently reads as "to the solve..."

File: src/pages/open-source-health-design.js [hunk 3]
```diff
@@ -0,0 +1,949 @@
...
+            </p>
+          </div>
+
+          <div className="pure-g container--justify-center ">
+            <div className="pure-u-lg-1-3">
+              <p>
+                <span style={numbersStyle}>$1.8M</span>
+                <span style={npStyle}>
+                  Studio internal investment <br /> in open source projects
```
Comment (ericbenwa) [inline src/pages/open-source-health-design.js:235]: Studio internal investment ==> GoInvo investment ... ?

File: src/pages/open-source-health-design.js [hunk 4]
```diff
@@ -0,0 +1,949 @@
...
+              <p>
+                <span style={numbersStyle}>$2.5M</span>
+                <span style={npStyle}>
+                  Clients investment <br /> in open source projects
+                </span>
+              </p>
+            </div>
+            <div className="pure-u-lg-1-3">
+              <p>
+                <span style={numbersStyle}>65+</span>
```
Comment (ericbenwa) [inline src/pages/open-source-health-design.js:249]: 60 or 65?
Stat above is over 60 and here is 65+

File: src/pages/open-source-health-design.js [hunk 5]
```diff
@@ -0,0 +1,949 @@
...
+          </div>
+
+          <ClientLogos openSource="true" />
+
+          <div className="content-padding pad-vertical" style={bgStyle}>
+            <h2 style={headerStyle}>What is Open Source Health Design?</h2>
+            <p style={pStyle}>
+              Open Source Health Design is a benevolent initiative to design and
+              build products aligned with the goals set by the US Department of
+              Health and Human Services. We design and build orolects with
```
Comment (ericbenwa) [inline src/pages/open-source-health-design.js:539]: "orolects" ? projects?

File: src/pages/open-source-health-design.js [hunk 6]
```diff
@@ -0,0 +1,949 @@
...
+              partners who commit to better healthcare and invest in open
+              source. This means making usable, useful and delightful designs
+              available to benefit communities and society as a whole
```
Comment (ericbenwa) [inline src/pages/open-source-health-design.js:542]: insert period

File: src/pages/open-source-health-design.js [hunk 7]
```diff
@@ -0,0 +1,949 @@
...
+                <div
+                  className="pure-u-lg-1-2 "
+                  style={{ paddingLeft: 40, paddingRight: 40 }}
+                >
+                  <h4 style={shStyle}>Living Health Lab</h4>
+                  <p style={pStyle}>
+                    Living Health Lab is a tool that helps people define and
+                    explore their personal health questions by tracking and
+                    visualizing their data with the support of a health buddy so
+                    that they can live their healthiest life
```
Comment (ericbenwa) [inline src/pages/open-source-health-design.js:691]: insert period

File: src/pages/open-source-health-design.js [hunk 8]
```diff
@@ -0,0 +1,949 @@
...
+                  className="pure-u-lg-1-2"
+                  style={{
+                    paddingLeft: 40,
+                    paddingRight: 40,
+                    paddingBottom: '5%',
+                  }}
+                >
+                  <h4 style={shStyle}>Health Manager</h4>
+                  <p style={pStyle}>
+                    Rosie is an open source Health Manager to support user
```
Comment (ericbenwa) [inline src/pages/open-source-health-design.js:731]: user => people

File: src/pages/open-source-health-design.js [hunk 9]
```diff
@@ -0,0 +1,949 @@
...
+                    aggregate all their health data into a single location and
+                    share that data in their community to meet their health
```
Comment (ericbenwa) [inline src/pages/open-source-health-design.js:733]: community => care team ?

File: src/pages/open-source-health-design.js [hunk 10]
```diff
@@ -0,0 +1,949 @@
...
+                  className="pure-u-lg-1-2"
+                  style={{ paddingLeft: 40, paddingRight: 40 }}
+                >
+                  <h4 style={shStyle}>Health Picture</h4>
+                  <p style={pStyle}>
+                    Discover our open source library for visualizing health
+                    data. Use hGraph for monitoring your patient population.
+                    Give patients a beautiful view of their health.
+                  </p>
+                  {/* <a>Learn more about the project</a> */}
```
Comment (ericbenwa) [inline src/pages/open-source-health-design.js:764]: Should this link to hGraph?

File: src/pages/open-source-health-design.js [hunk 11]
```diff
@@ -0,0 +1,949 @@
...
+                    paddingRight: 40,
+                    paddingBottom: '5%',
+                  }}
+                >
+                  <h4 style={shStyle}>Health Data Ownership</h4>
+                  <p style={pStyle}>
+                    We need patient data ownership rights to protect patient
+                    care. We've mapped out the needs for owning your health data
+                    and have illustrated them to be digestible for policy makers
+                    and the general public
```
Comment (ericbenwa) [inline src/pages/open-source-health-design.js:805]: insert period

File: src/pages/open-source-health-design.js [hunk 12]
```diff
@@ -0,0 +1,949 @@
...
+
+          {/* CTA for the community  */}
+          <div
+            className="pure-u-1 content-padding pad-vertical"
+            style={bgStyle}
+          >
+            <div className="pure-u-lg-1-2 ">
+              <h2 style={headerStyle}>Let’s build better care together!</h2>
+              <p style={pStyle}>
+                We’re a growing communtiy of healthcare professionals, designers
```
Comment (ericbenwa) [inline src/pages/open-source-health-design.js:895]: typo "community"

File: src/pages/open-source-health-design.js [hunk 13]
```diff
@@ -0,0 +1,949 @@
...
+                and engineers. Subscribe to our Open Source Health Design
+                newsletter where we regularly share updates on projects and
+                organize online event giving you the opportunity to connect with
+                a diverse network, ask questions, show your work and exchange
+                ideas.
```
Comment (ericbenwa) [inline src/pages/open-source-health-design.js:900]: "Subscribe to our Open Source Health Design newsletter where we regularly share updates on projects and organize online events giving you the opportunity to connect with a diverse network, ask questions, show your work, and exchange ideas."

tweaks on plural for events and added comma

**Comment by ericbenwa** [comment]
Link to LHL is going to the homepage. I know LHL feature isn't live but can we guesstimate its URL and drop in now?

Rosie is an open source health manager to support people aggregate all their health data...
("to support people aggregate" sounds a little off)
=? a suggestion ("to support people aggregating")

"Here is the debut issue of our Open Source Health Journal, advocating open source ideas to change healthcare for the better."
(Since we only have one, with a second not coming in short-term, do we rewrite to remove the word "Debut"?)

File: src/pages/open-source-health-design.js [hunk 14]
```diff
@@ -0,0 +1,537 @@
...
+                        health goals.
+                        <br />
+                      </p>
+                    </div>
+                  </div>
+                </Card>
+              </div>
+
+              <div className="pure-u-1">
+                <Card link="https://www.goinvo.com/work/hgraph">
```
Comment (craigmcginley) [inline src/pages/open-source-health-design.js:363]: I think for "internal" links, aka ones going to somewhere within www.goinvo.com we should just use the path `link="/work/hgraph"`

For links going elsewhere:
`link="https://www.google.com/" externalLink={true}`

Doing this helps with page load performance, and can also be used for special navigation techniques (which I don't think we employ on our site)

File: src/pages/open-source-health-design.js [hunk 15]
```diff
@@ -0,0 +1,537 @@
...
+                  sizes={config.sizes.fullToHalfAtLargeInsideMaxWidth}
+                />
+                <p className="text">
+                  Invest in projects in public health and pioneer cutting-edge
+                  design for the common good.
+                </p>
+              </div>
+
+              <div className="pure-u-1 card" style={{ marginTop: '7%' }}>
+                <Card link="/">
```
Comment (craigmcginley) [inline src/pages/open-source-health-design.js:306]: you can add `/vision/living-health-lab` here and it will be merged soon

**Review by craigmcginley — CHANGES_REQUESTED** [review]

**Review by craigmcginley — APPROVED** [review]

**Review by craigmcginley — APPROVED** [review]

**Pull request merged**